Makefile

```diff
@@ -1,6 +1,6 @@
 PAKAGES = freetype2 harfbuzz fribidi
 
-CFLAGS = `pkg-config --cflags $(PAKAGES)` -DTESTING -Wall -ansi 
+CFLAGS = `pkg-config --cflags $(PAKAGES)` -DTESTING -std=c99 -Wall
```

Comment:
It would be nice to have all the warnings you were compiling with enable here, to avoid there recurrence in the future.